impeller/compiler/runtime_stage_data.cc

```diff
@@ -63,10 +59,6 @@ static std::optional<uint32_t> ToJsonStage(spv::ExecutionModel stage) {
       return 1;  // fb::Stage::kFragment;
     case spv::ExecutionModel::ExecutionModelGLCompute:
       return 2;  // fb::Stage::kCompute;
-    case spv::ExecutionModel::ExecutionModelTessellationControl:
```

Comment:
Unrelated to this patch: This is a bit of an odd API, can this just return `std::optional<spv::ExecutionModel>` instead of the desugared int. Then the caller can cast it to an `int`. That way, we don't have to match `fb::Stage::kVertex` to `0` and so on. In case we change the .fbs to change the order, we won't know we broke this here.

The `ToStage` method above does this correctly IMO.

Reply:
Done